src/builder.rs

```diff
@@ -193,6 +193,11 @@ impl Builder {
             panic!("Maximum message latency must be greater than minimum.");
         }
 
+        if self.config.tick.as_nanos() % Duration::from_millis(1).as_nanos() != 0  {
```

Comment:
Would it be simpler to just check `< Duration::from_millis(1)`?

Reply:
I think someone could provide a value that was > 1 ms, but had a higher resolution, e.g. `Duration::from_micros(1500)`, which I believe we would want to disallow still given that comment on `tokio::time::sleep`.

Though, we might want to disallow `Duration::ZERO` in addition I suppose?

Reply:
I don't think it needs to be this restrictive. Given that the runtime is paused we aren't actually running for the duration, we are simply advancing time by it.

`Duration::ZERO` is a good callout, and I'm not actually sure what would happen in that case. My guess is that nothing runs.